src/openforms/registrations/contrib/objects_api/registration_variables.py

```diff
@@ -26,6 +26,17 @@ class Registry(BaseRegistry[BaseStaticVariable]):
 """The Objects API registration variables registry."""
 
 
+@register("public_reference")
+class PublicReference(BaseStaticVariable):
```

Comment:
I don't see this variable mentioned in the `Vaste variabelen` section, should this be mentioned there? Or is this mentioned in a specific page for Objects API registration?

Reply:
it's not a static variable, but a registration variable and visible in the UI when you select an objects API registration backend :) and there the title + type are displayed (it's "self-documenting" via the registry).